app/models/work_type.rb

```diff
@@ -135,7 +135,7 @@ def self.all
     [
       new(id: 'data', label: 'Data', icon: 'chart-bar', subtypes: DATA_TYPES),
       new(id: 'image', label: 'Image', icon: 'images', subtypes: IMAGE_TYPES),
-      new(id: 'mixed material', label: 'Multimedia', icon: 'play', subtypes: MIXED_TYPES),
+      new(id: 'mixed materials', label: 'Mixed Materials', icon: 'play', subtypes: MIXED_TYPES),
```

Comment:
Please don't change the id, that's a MODS term: https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/8d8851203a026dec1eff8021cc14ec43805cdfa0/mods_cocina_mappings/mods_to_cocina_typeOfResource.txt#L67

Reply:
Reverted in latest commit.